yazilim-dilleri/c/test.c

```diff
@@ -0,0 +1,6 @@
+#include <stdio.h>
+
+int main(){
+   printf("Hello World");
+   return 0;
+}
```

Comment:
```suggestion
int main(void){
   printf("Hello World");
   return 0;
}
```
Daha uygun olabilir.
https://www.geeksforgeeks.org/difference-int-main-int-mainvoid/